dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/DLCWallet.scala

```diff
@@ -890,7 +969,10 @@ abstract class DLCWallet extends Wallet with AnyDLCHDWalletApi {
 
     executorFromDb(dlcDb, dlcOffer, dlcAccept, fundingInputs)
       .flatMap { executor =>
-        val outcomeSigs = outcomeSigDbs.map(_.toTuple).toMap
+        val outcomeSigs = outcomeSigDbs
+          .filter(_.isInitiator == !dlcDb.isInitiator)
```

Comment:
This deserves a comment, what is the plain english explaination of what's going on here?